test/validation/org/apache/cassandra/bridges/ctoolbridge/CToolBridge.java

```diff
@@ -137,9 +140,9 @@ public String readClusterLogs(String testName)
         String combinedResult = "";
         String existingFolder = CASSANDRA_DIR + "/build/test/logs/validation/ctoolbridge/" + testName;
 
-        if(ArchiveClusterLogs.checkForFolder(existingFolder))
+        if (ArchiveClusterLogs.checkForFolder(existingFolder))
         {
-            for(int count = 1; count <= nodeCount; count++)
+            for (int count = 1; count <= nodeCount; count++)
```

Comment:
Yeah.. I am naming the log files starting from node1.log and not node0.log. So this is in accordance to that.